tests/tests/test-block.ts

```diff
@@ -13,25 +13,24 @@ describeWithMoonbeam("Moonbeam RPC (Block)", `simple-specs.json`, (context) => {
 		expect(await context.web3.eth.getBlockNumber()).to.equal(0);
 	});
 
-	it.skip("should return genesis block", async function () {
+	it("should return genesis block", async function () {
 		expect(await context.web3.eth.getBlockNumber()).to.equal(0);
-
 		const block = await context.web3.eth.getBlock(0);
 		expect(block).to.include({
-			author: "0x0000000000000000000000000000001234567890",
+			author: "0x0000000000000000000000000000000000000000",
```

Comment:
Why is the author 0 ?
Is it only for the Genesis block or all the blocks will have author 0 ?

Reply:
Apparently for all.

Reply:
So, it could be actually the same problem. There is no `palletAura` in the `simple-specs.json`, so Aura genesis state is not initialized.

Reply:
It was hardcoded in the runtime, to return that number

Reply:
I don't see it hardcoded around here https://github.com/PureStake/moonbeam/blob/master/runtime/src/standalone.rs#L59

It looks like we are using the actual aura author address in the standalone node. And in manual seal mode, that aura info is forged. Not surprising it returns `0` as it does it the Frontier node.